test/plugins/validation/2and3/schema-ibm.js

```diff
@@ -874,12 +1118,11 @@ describe('validation plugin - semantic - schema-ibm - OpenAPI 3', () => {
     const res = validate({ jsSpec: spec, isOAS3: true }, config);
     expect(res.errors.length).toEqual(2);
     expect(res.errors[0].message).toEqual(
-      'Property type+format is not well-defined.'
+      'Invalid type. Valid types are: integer, number, string, boolean, object, array.'
     );
     expect(res.errors[1].message).toEqual(
-      'Property type+format is not well-defined.'
+      'Schema of type string should use one of the following formats: byte, binary, date, date-time, password.'
     );
-    expect(res.warnings.length).toEqual(0);
```

Comment:
Why remove this line?

Reply:
On some of my previous PRs, we talked about removing the additional checks that were not directly related to the test. When I've seen this in other tests, I've removed the checks to prevent unexpected test failures in the future due to adding a validation rule to a module.

Reply:
I can't argue with that. Carry on 🙂